src/view/observer/focusobserver.js

```diff
@@ -8,6 +8,10 @@
  */
 
 import DomEventObserver from './domeventobserver';
+import global from '@ckeditor/ckeditor5-utils/src/dom/global';
+
+const setTimeout = global.window.setTimeout;
+const clearTimeout = global.window.clearTimeout;
```

Comment:
I mean: tests which ckecks if `setTimeout`/`clearTimeout` are called are too "unit" for me. They should check if `render` is called when it should be and is not called when it should not.